src/internal/pfsdb/model.go

```diff
@@ -36,6 +36,7 @@ type ProjectRow struct {
 	Description string                `db:"description"`
 	Metadata    pgjsontypes.StringMap `db:"metadata"`
 	CreatedAtUpdatedAt
+	CreatedBy *string `db:"created_by"`
```

Comment:
It needs to be nullable because not every project has a creator. Projects created before tracking do not. I think that’s preferable to a sentinel principal with an empty string name.